symengine/tests/basic/test_cse.cpp

```diff
@@ -1,19 +1,25 @@
 #include "catch.hpp"
+#include "symengine/dict.h"
```

Comment:
Use `<` instead of `"`

Reply:
thanks, my IDE seem to have "helped" me here... dropping those includes.